notebooks/official/matching_engine/similar_products_recommender_using_matching_engine.ipynb

```diff
@@ -0,0 +1,1683 @@
+{
```

Comment:
<h1>Text-Based Product Embeddings -&gt; lowercase</h1><p> </p><p>Add missing Colab link</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Please find that notebook cannot be run in colab, as this notebook has to be executed in a notebook which is created in a vpc network created in this notebook. It has dependency on vpc network created in this notebook.</p>